<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>material_site_editor</class>
 <widget class="QWidget" name="material_site_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>502</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Overlay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDoubleSpinBox:disabled {background-color: LightGray;}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="site_atom_types_group">
     <property name="title">
      <string>Site Atom Types</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1" colspan="2">
       <widget class="QTableWidget" name="table">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Symbol</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Charge</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Occupancy</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>U</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="remove_atom_type">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="select_atom_types">
        <property name="text">
         <string>Select Atom Types</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="site_settings_group">
     <property name="title">
      <string>Site Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1" colspan="3">
       <widget class="ScientificDoubleSpinBox" name="total_occupancy">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScientificDoubleSpinBox" name="coords_x">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="thermal_factor_label">
        <property name="text">
         <string>Thermal Factor Type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="ScientificDoubleSpinBox" name="coords_y">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="ScientificDoubleSpinBox" name="coords_z">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="total_occupancy_label">
        <property name="text">
         <string>Total Occupancy:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fractional_coordinates_label">
        <property name="text">
         <string>Fractional Coords:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="thermal_factor_type">
        <item>
         <property name="text">
          <string>U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.py</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>total_occupancy</tabstop>
  <tabstop>coords_x</tabstop>
  <tabstop>coords_y</tabstop>
  <tabstop>coords_z</tabstop>
  <tabstop>thermal_factor_type</tabstop>
  <tabstop>table</tabstop>
  <tabstop>select_atom_types</tabstop>
  <tabstop>remove_atom_type</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
